tests/test_core.py

```diff
@@ -46,6 +46,19 @@ def test_init_df_with_index(test_pd_df):
     pd.testing.assert_frame_equal(df.timeseries().reset_index(), test_pd_df)
 
 
+def test_init_from_iamdf(test_df_year):
```

Comment:
Do you have a test for what happens if there are kwargs?

Reply:
added a test!